src/main/world/creature/creature-sfx.gd

```diff
@@ -88,25 +122,20 @@ func play_door_chime(delay: float = -1) -> void:
 	if delay < 0:
 		delay = CHIME_DELAYS[randi() % CHIME_DELAYS.size()]
 	yield(get_tree().create_timer(delay), "timeout")
-	chime_sounds[randi() % chime_sounds.size()].play()
-	yield(get_tree().create_timer(0.5), "timeout")
-	play_hello_voice()
-
-
-"""
-Plays a voice sample, interrupting any other voice samples which are currently playing for this specific creature.
-"""
-func _play_voice(audio_stream: AudioStreamPlayer2D) -> void:
-	if _current_voice_stream and _current_voice_stream.playing:
-		_current_voice_stream.stop()
-	audio_stream.play()
-	_current_voice_stream = _combo_voices[_combo_voice_index]
+	
+	if is_inside_tree():
+		$DoorChime.stream = _chime_sounds[randi() % _chime_sounds.size()]
+		$DoorChime.play()
+		yield(get_tree().create_timer(0.5), "timeout")
+	
+	if is_inside_tree():
+		play_hello_voice()
 
 
 func _on_Creature_food_eaten() -> void:
-	var munch_sound: AudioStreamPlayer2D = _munch_sounds[randi() % _munch_sounds.size()]
-	munch_sound.pitch_scale = rand_range(0.96, 1.04)
-	_play_voice(munch_sound)
+	$Munch.stream = _munch_sounds[randi() % _munch_sounds.size()]
+	$Munch.pitch_scale = rand_range(0.96, 1.04)
+	$Munch.play()
```

Comment:
This is worth documenting in the commit comment, and possibly in a code comment as well.

Reply:
Added code comment, commit comment